tests/unit/server/mvc/models/UserTest.js

```diff
@@ -0,0 +1,95 @@
+var path = require("path");
+
+var bogusThinkyConfig = {db: "bogus"};
+var Models = {};
+
+module.exports = {
+  setUp: function(callback){
+    Models.User = require(path.normalize(__dirname + "/../../../../../server/mvc/models/User.js"));
+    var bogusThinky = require("thinky")(bogusThinkyConfig);
+    Models.User.init(bogusThinky);
+    callback();
+  },
+  tearDown: function(callback){
+    var bogusThinky = require("thinky")(bogusThinkyConfig);
+    bogusThinky.r.table("users").delete().run().finally(callback);
+  },
+  "CanFindByUserId": function(test) {
```

Comment:
I guess we can use spaces instead of camel-casing it.

I don't think there's a `describe` feature on the tests themselves. Only on the asserts.